index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Word</title>
+    <link rel="stylesheet" href="/css/style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
+</body>
+</html>
```

Comment:
Good job :+1: . However, it will be nicer if you could add a descriptive README file to your project. That file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.